Seed file

german-e-mas opened the pull request:
In this PR I update the seed file, in order to create initial content for the database. The first article has no comments, while the second has three.

I feel like there is way to make this better or at least more efficient...

ptal @basicNew

german-e-mas commented on db/seeds.rb:
```diff
@@ -5,3 +5,10 @@
+Article.create(title: 'First Article', text: 'This is the body of the First Article in the blog.\nThis article starts without any comment.')
+article = Article.create(title: 'Second Article', text: 'This article has three different comments.')
+
+Comment.create(commenter: 'HNDK', body: 'First comment!', article_id: article.id)
```
This works because Comments **belong to** Articles, right? I tried removing the relationship and passing the article this way and I get exceptions thrown.

german-e-mas commented:
I addressed the comments. Included the `!` in the methods and also passed the article as a parameter instead of just it's ID.

ptal @basicNew

basicNew commented:
Great! One more thing: ideally, seeds should be idempotent, so if an example already exists you don't override it. See how you can rewrite the current seed so that if you run it twice it generates the articles and comments only once (hint: look at `find_or_create_by` and/or `first_or_create`)

german-e-mas commented:
I addressed the comment about using `find_or_create_by`. I found that many use only one argument, but I tested in the console and works well, creating things only if they don't exist in the database, taking all the arguments into account.

ptal @basicNew

basicNew commented:
Great, this LGTM @german-e-mas !

german-e-mas commented:
Thanks for the review and the advices @basicNew . I'll proceed to merge.
